nameservice/errors.go

```diff
@@ -61,3 +78,18 @@ func IsInvalidAddressError(err error) bool {
 	}
 	return false
 }
+
+func IsRfc1035Error(err error) bool {
+	_, ok := Rfc1035StatusMap[err]
+	return ok
+}
+
+func wrapError(err error) error {
+	if err == nil {
+		return ErrNon
```

Comment:
Best just to return `nil` here as then downstream it can be dealt with as a nil error the same as others